flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnCLI.java

```diff
@@ -85,6 +89,10 @@ public FlinkYarnCLI(String shortPrefix, String longPrefix) {
 		DETACHED = new Option(shortPrefix + "a", longPrefix + "attached", false, "Start attached");
 		ZOOKEEPER_NAMESPACE = new Option(shortPrefix + "z", longPrefix + "zookeeperNamespace", true, "Namespace to create the Zookeeper sub-paths for high availability mode");
 
+		LIB_JARS = new Option(shortPrefix + "libjars", longPrefix + "libjars", true, "Jar file paths for job, like /home/user/lib/test.jar");
+		FILES = new Option(shortPrefix + "files", longPrefix + "files", true, "Normal file paths for job, like /home/user/lib/test.dict");
+		ARCHIVES = new Option(shortPrefix + "archives", longPrefix + "archives", true, "Archived file uris for job, like hdfs:///users/flink/common_dict#dict");
```

Comment:
It is totally different using the BlobStore. For clusters with BlobStore, DistributedCache may be enough. I think in the future, we can support DistributedCache with BlobStore in the future, and support external BlobStore structure which already run on existed framework, like yarn distributed cache in the future